x/concentrated-liquidity/fuzz_test.go

```diff
@@ -40,7 +40,7 @@ type positionAndLiquidity struct {
 }
 
 func TestFuzz_Many(t *testing.T) {
-	fuzz(t, defaultNumSwaps, defaultNumPositions, 100)
+	fuzz(t, defaultNumSwaps, defaultNumPositions, 10)
```

Comment:
Driveby change, now that were through CL, running 100 iterations on every save no longer make sense. (It also makes test on save expensive!)